app/services/update/update_client_service.py

```diff
@@ -234,8 +236,9 @@ def __clear_and_add_nodes(self, updated_nodes: List[Node]) -> None:
                 node.clear_for_cash()
                 self.__cache.add_node(node)
 
+
     def update_page(
-        self, entries: List[BundleEntry], adjacency_map_service: AdjacencyMapService
+        self, entries: List[BundleEntry], adjacency_map_service: AdjacencyMapService, origin_url: str
```

Comment:
Can be removed as it is unused
```suggestion
        self, entries: List[BundleEntry], adjacency_map_service: AdjacencyMapService
```